Trim Dynamo shards 7->3

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #117869

We added all of the tests we wanted for now. These fit comfortably in 3
shards (the total test time previously was 0.5 hours on each shard).
Going to decrease the number of shards to 3 so that it's less unwieldy
to work with.

zou3519 committed:
Trim Dynamo shards 7->3

We added all of the tests we wanted for now. These fit comfortably in 3
shards (the total test time previously was 0.5 hours on each shard).
Going to decrease the number of shards to 3 so that it's less unwieldy
to work with.

[ghstack-poisoned]

huydhn reviewed (approved):
LGTM!  Let's see what the duration of the 3 shards are

zou3519 committed:
Update on "Trim Dynamo shards 7->3"


We added all of the tests we wanted for now. These fit comfortably in 3
shards (the total test time previously was 0.5 hours on each shard).
Going to decrease the number of shards to 3 so that it's less unwieldy
to work with.

[ghstack-poisoned]

zou3519 commented:
Dynamo shards completed. Timings are roughly 1 hour / shard, which looks good.

zou3519 commented:
@pytorchbot merge -f "dynamo CI passed"
